test/boost/sstable_mutation_test.cc

```diff
@@ -64,6 +63,7 @@ SEASTAR_THREAD_TEST_CASE(nonexistent_key) {
             });
         });
     }).get();
+  }).get();
```

Comment:
nit: why thread within a thread?
can change the test definition to `SEASTAR_TEST_CASE` and return `test_env::do_with_async` instead of `get()`ing it.

Reply:
To have a minimum change. The extra 128k allocation doesn't bother me for a test.